src/server/common/helpers/token/token-utils.test.js

```diff
@@ -0,0 +1,42 @@
+import { createToken, secrets } from './token-utils.js'
+import { token } from '@hapi/jwt'
+
+jest.mock('@hapi/jwt', () => ({
+  token: {
+    generate: jest.fn()
+  }
+}))
+
+describe('token-utils', () => {
+  const apiKey =
+    'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
+  const secret = apiKey.substring(apiKey.length - 36, apiKey.length)
+  const iss = apiKey.substring(apiKey.length - 73, apiKey.length - 37)
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('should generate a token with correct payload and options', () => {
+    const iat = Math.round(Date.now() / 1000)
```

Comment:
This'll leave our tests with a chance of intermittent failure, since the next time Date.now() is called (inside the function under test), the date may have ticked over. 

Mock?